File: src/Vincent OS/Login2bureau.cs
using System;

namespace Vincent_OS
{
    public partial class Login2bureau
    {
        public Login2bureau()
        {
            InitializeComponent();
        }

        private void Login2bureau_Load(object sender, EventArgs e)
        {
            Timer1.Stop();
            ProgressBar1.Value = 0;
            Timer1.Start();
        }

        private void Timer1_Tick(object sender, EventArgs e)
        {
            ProgressBar1.Increment(3);
            if (ProgressBar1.Value == 100)
            {
                Timer1.Stop();
                ProgressBar1.Value = 0;
                My.MyProject.Forms.Bureau.Show();
                Close();
                GC.Collect();
                GC.WaitForPendingFinalizers();
            }
        }
    }
}

File: src/Vincent OS/Login2bureau.vb
Public Class Login2bureau

    Private Sub Login2bureau_Load(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles MyBase.Load
        Timer1.Stop()
        ProgressBar1.Value = 0
        Timer1.Start()
    End Sub

    Private Sub Timer1_Tick(ByVal sender As System.Object, ByVal e As System.EventArgs) Handles Timer1.Tick
        ProgressBar1.Increment(3)
        If ProgressBar1.Value = 100 Then
            Timer1.Stop()
            ProgressBar1.Value = 0
            Bureau.Show()
            Me.Close()
        End If
    End Sub
End Class